src/content/image.js

```diff
@@ -0,0 +1,65 @@
+import bind from './bind';
+import { width as A4Width, height as A4Height } from '../boxes/a4';
+
+const number = n => {
+  if (n > -1e21 && n < 1e21) {
+    return Math.round(n * 1e6) / 1e6;
+  }
+  throw new Error(`unsupported number: ${n}`);
+};
+
+const transform = (m11, m12, m21, m22, dx, dy) => {
+  const matrix = [1, 0, 0, 1, 0, 0];
+  const [m0, m1, m2, m3, m4, m5] = matrix;
+
+  matrix[0] = m0 * m11 + m2 * m12;
+  matrix[1] = m1 * m11 + m3 * m12;
+  matrix[2] = m0 * m21 + m2 * m22;
+  matrix[3] = m1 * m21 + m3 * m22;
+  matrix[4] = m0 * dx + m2 * dy + m4;
+  matrix[5] = m1 * dx + m3 * dy + m5;
+
+  const values = [m11, m12, m21, m22, dx, dy].map(v => number(v)).join(' ');
+  return `${values} cm`;
+};
+
+const getTransformation = (size) => {
+  const { width, height } = size;
+  const [bw, bh] = [A4Width, A4Height];
```

Comment:
Not so happy that we hardcode to A4 size?

Cannot this be done in the layouter. What did you get as width and height if you just set it to 100%? or do you loose the relation then?